services/storage/tests/unit/test_dsm_dsmcleaner.py

```diff
@@ -35,6 +36,8 @@
 pytest_simcore_core_services_selection = ["postgres"]
 pytest_simcore_ops_services_selection = ["adminer"]
 
+_Faker: Faker = Faker()
```

Comment:
Either use `_faker` or `_FAKER`. That one is not a python convention.
```suggestion
_faker: Faker = Faker()
```

Reply:
Fixed